projectq/ops/_gates.py

```diff
@@ -205,6 +217,9 @@ def matrix(self):
         return np.matrix([[cmath.exp(1j * self.angle), 0],
                           [0, cmath.exp(1j * self.angle)]])
 
+    def __str__(self):
```

Comment:
Why is this change needed? I would rather keep the old way of printing the phase gate as `"Ph(angle)"`.

Reply:
I am sorry, I overlooked this while undoing changes I had made to the printing method.

Reply:
No worries!